README.md

```diff
@@ -123,32 +107,34 @@ You are first required to build the tests, then you can run them:
 1. To build the **Detox end to end tests**: `npm run detox-build`
 2. To run the **Detox end to end tests**: `npm run detox-test`
 
-## Troubleshooting
+# Troubleshooting
 1. If you are having issues with **_Getting Started_**, please reference [React Native's Documentation](https://reactnative.dev/docs/environment-setup)
-2. If you are running into issues communicating with `expensify.com.dev` (CORS, SSL, etc.), running via `ngrok` is recommended, see step 3 in **_Getting Started_**
+2. If you are running into issues communicating with the API please verify your `.env` file is set up correctly for the platform you are trying to run.
```

Comment:
NAB: Can we link to a header in the README that takes them back to `## Getting started` section that explains all this?

Reply:
Sure! But to clarify what specifically are you wanting to be added to the Getting Started section?

Reply:
Oh, yeah, that could have been clearer lol

I'm thinking we add a link on `set up correctly` that links back to the `Getting Started` section of this doc.

Reply:
I see your newest commit, and that works, but yeah, I was more thinking right in the middle of where it talks about what to do if you're having problems, here's a link of where to read again.